mobile/ios/ios-core.ts

```diff
@@ -593,6 +595,10 @@ export class MobileDevice implements Mobile.IMobileDevice {
 	public afcDirectoryCreate(afcConnection: NodeBuffer, path: string): number {
 		return this.mobileDeviceLibrary.AFCDirectoryCreate(afcConnection, path);
 	}
+	
+	public afcFileInfoOpen(afcConnection: NodeBuffer, path: string, afcDirectory: NodeBuffer): number {
```

Comment:
this method is not doing anything, is there a reason to have it here? The code which calls it, can call `this.mobileDeviceLibrary.AFCFileInfoOpen` directly.